CIP-1775/README.md

```diff
@@ -0,0 +1,108 @@
+---
+CIP: 1775
+Title: Delegated Spending Authorizations
+Authors: Ivan Mikushin <sorrel_cantos.0@icloud.com>
+Comments-URI:
+Status: Proposed
+Type: Standards Track
+Created: 2022-11-17
+License: CC-BY-4.0
+---
+
+# Delegated Spending Authorizations
+
+"Give me custody, or give me death!" - [@ArmySpies](https://twitter.com/ArmySpies/status/1592586212816355328)
+
+## Abstract
+
+We propose to allow users to delegate authorized spending of outputs in their wallets to smart contracts or other users, without sending said outputs out of their wallets.
+
+## Motivation
+
+In order to use any dApp on Cardano (e.g. a decentralized exchange or an NFT marketplace), a user has to **deposit tokens** to a script address, thereby **giving up custody** of the tokens.
+
+This can be problematic for a number of reasons.
+
+- The user loses control of the assets they send out to a smart-contract, unless the smart contract is specifically designed to allow for such control (e.g. giving a "Cancel trade" or "De-list NFT" button in the UI).
+- Assets sent out from a user's wallet:
+  - are no longer  **delegated** to the stake pool of the user's choice.
+  - cannot participate in **governance voting** on behalf of the user.
+- Any output, accidentally sent to a script address without a datum is effectively lost now (because of how transaction validation scripts are invoked).
+- Requiring users to deposit assets into a dapp may subject the dapp's owners to regulatory scrutiny.
+
+We believe, this is not strictly necessary and would like a way to allow users to delegate spending authorization for some outputs **within** their wallets to other users (smart contracts, multisigs, wallet addresses).
+
+### Use Cases
+
+Instead of sending tokens to the dapp smart-contract, the user creates an output in their own wallet that delegates spending authorization to a smart-contract (or other entity).
+
+Some **existing** use cases, where this can be beneficial:
+
+1. Swapping tokens on a decentralized exchange.
+2. Providing liquidity on a decentralized exchange.
+3. Listing an NFT on a marketplace.
+
+Some **New** use cases, enabled by this proposal:
+
+1. On-chain noncustodial asset management. A user can delegate managing (some of) their assets to a manager, another individual user or organization.
```

Comment:
This is achievable already with multi-sig. You could easily send funds to an address that multiple individuals are allowed to spend from. It's just a matter of properly showing these funds in a wallet interface.

The funds could remain fully controllable by you and fully controllable by the other individual or organization. You could do a 1 of 2 multisig to achieve it.

Reply:
Cool. Can I vote in Catalyst with this multisig? How do I delegate to an SPO with the multisig?

Reply:
Yes, the staking credential and the spending credential are entirely independent, which is a very important point for this proposal!

Reply:
Updated the text.